calculator.js

```diff
@@ -1,5 +1,112 @@
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
+// getting colors on the command lines
+const colors = require('colors');  
+// getting user input
+const prompt = require('prompt');
+
+const operationOptions = {
+  add: '+',
+  subtract: '-',
+  divide: '/',
+  multiply: '*',
+  modulo: '%',
+  exponents: "^",
 }
+
+//helper function to put greeting and choices on the command line for user
+const greeting = () => {
+  console.log("Welcome to JS calculator! Let's do some math! Please enter two numbers and an operation. \nchoices for operation are below:".blue);
+  console.log("=======================================================");
+  for(let key in operationOptions){
+    console.log(key, "or", operationOptions[key]);
+  };
+  console.log("=======================================================");
+}
+
+
+// helper function to check if the numbers provided are valid
+const checkNumber = (input) => {
+  return (!isNaN(input.num1) && !isNaN(input.num2));
+};
+
+
+// helper function to check if the operation provided is valid
+const checkOperation = (input) => {
+  let operationArray = ["add", "+", "subtract", "-", "divide", "/", "multiply", "*", "modulo", "%", "^", "exponents" ]
```

Comment:
You should prefer `const` when declaring variables and only use `let` if you need to reassign.